server/main.py

```diff
@@ -129,7 +131,9 @@ async def index(request):
         job.command(f'git merge-base --is-ancestor {_shell_escape(commit)} HEAD')
         job.command(f'git checkout {_shell_escape(commit)}')
         # Make sure the file is in the repository.
-        script_file = script.partition(' ')[0]
+        if not isinstance(script, list):
+            script = str(script).split(' ')
+        script_file = script[0]
```

Comment:
It's not obvious from the code that the `script` parameter can contain command-separated arguments. Maybe could move the parsing to 109, and make it like this?

```
script = params['script']
if not script:
    raise web.HTTPBadRequest(reason='Invalid script parameter')
if isinstance(script, str):
    script = script.split(' ')
script_file, *script_args = script
```

Plus, I'd split on any whitespace (`script.split(' ')` -> `script.split()`) to allow for multiple spaces.